packages/calcite-components/src/components/flow-item/flow-item.e2e.ts

```diff
@@ -172,4 +206,22 @@ describe("calcite-flow-item", () => {
 
     expect(scrollSpy).toHaveReceivedEventTimes(1);
   });
+
+  it("honors calciteFlowItemToggle event", async () => {
+    const page = await newE2EPage({
+      html: "<calcite-flow-item collapsible>test</calcite-flow-item>",
+    });
+
+    const toggleSpy = await page.spyOnEvent("calciteFlowItemToggle");
+
+    await page.evaluate(() => {
```

Comment:
It might be possible to this entirely w/ E2E APIs. Could you try the following?

```tsx
const panel = await page.find("calcite-flow-item >>> calcite-panel");
await panel.triggerEvent("calcitePanelToggle");
```

Reply:
Yep works! 💥